map/src/map/markers/MarkerIcon.jsx

```diff
@@ -0,0 +1,19 @@
+import L from "leaflet";
```

Comment:
Looks like Separate file is not needed, so it could be combined with MarkerOptions into 1 like 'CircleMarker'